lib/syscontrol/syscontrol.go

```diff
@@ -71,20 +72,21 @@ curl -i -XPOST 'http://127.0.0.1:8086/debug/ctrl?mod=force_broadcast_query&enabl
 */
 
 const (
-	DataFlush           = "flush"
-	DownSampleInOrder   = "downsample_in_order"
-	compactionEn        = "compen"
-	compmerge           = "merge"
-	snapshot            = "snapshot"
-	ChunkReaderParallel = "chunk_reader_parallel"
-	BinaryTreeMerge     = "binary_tree_merge"
-	PrintLogicalPlan    = "print_logical_plan"
-	SlidingWindowPushUp = "sliding_window_push_up"
-	ForceBroadcastQuery = "force_broadcast_query"
-	Failpoint           = "failpoint"
-	Readonly            = "readonly"
-	LogRows             = "log_rows"
-	verifyNode          = "verifynode"
+	DataFlush            = "flush"
+	DownSampleInOrder    = "downsample_in_order"
+	compactionEn         = "compen"
+	compmerge            = "merge"
+	snapshot             = "snapshot"
+	ChunkReaderParallel  = "chunk_reader_parallel"
+	BinaryTreeMerge      = "binary_tree_merge"
+	PrintLogicalPlan     = "print_logical_plan"
+	SlidingWindowPushUp  = "sliding_window_push_up"
+	ForceBroadcastQuery  = "force_broadcast_query"
+	Failpoint            = "failpoint"
+	Readonly             = "readonly"
+	LogRows              = "log_rows"
+	verifyNode           = "verifynode"
+	TimeFilterProtection = "time-filter-protection"
```

Comment:
The naming style of the time-filter-protection variable is different from that of other variables. The hyphen (-) is changed to underscore (_).

Reply:
> The naming style of the time-filter-protection variable is different from that of other variables. The hyphen (-) is changed to underscore (_).
ok